spec/requests/subscriptions/create_spec.rb

```diff
@@ -17,7 +17,7 @@
 
   context 'when api payment method fails' do
```

Comment:
I want to see contexts here for when we have affiliate tag and for when we don't ;)

The same for `spec/requests/users/sign_up_spec.rb`. Now that we have a parameter that will influence business logic (more than first and last name) I think is time we check if all parameters are being saved in a test.

Reply:
I'm having trouble setting up the `request.env['affiliate.tag']` value for this test. Any suggestions?